apis/hive/v1/clusterpool_types.go

```diff
@@ -218,8 +218,11 @@ const (
 	// ClusterPoolAllClustersCurrentCondition indicates whether all unassigned (installing or ready)
 	// ClusterDeployments in the pool match the current configuration of the ClusterPool.
 	ClusterPoolAllClustersCurrentCondition ClusterPoolConditionType = "AllClustersCurrent"
-	// ClusterPoolInventoryValidCondition is set to provide information on whether the cluster pool inventory is valid
+	// ClusterPoolInventoryValidCondition is set to provide information on whether the cluster pool inventory is valid.
 	ClusterPoolInventoryValidCondition ClusterPoolConditionType = "InventoryValid"
+	// ClusterPoolDeletionPossibleCondition gives information about a deleted ClusterPool which is pending cleanup.
+	// Note that it is normal for this condition to remain uninitialized until the ClusterPool is deleted.
```

Comment:
Yeah, might want to remove this comment to avoid confusion. The way you've set it up, we are always initializing this condition.

Reply:
Done.